src/app/desktop/windows/terminal/terminal-states.ts

```diff
@@ -1500,9 +1607,9 @@ export class BruteforceTerminalState extends ChoiceTerminalState {
   };
 
   constructor(terminal: TerminalAPI,
-              private domSanitizer: DomSanitizer,
-              private callback: (response: boolean) => void,
-              private startSeconds: number = 0) {
+    private domSanitizer: DomSanitizer,
+    private callback: (response: boolean) => void,
+    private startSeconds: number = 0) {
```

Comment:
Please do not change indentations where you have not changed the code.

Reply:
Sorry I have Problems with VS Code

Reply:
I'll fix it for you